scripts/run_tests.sh

```diff
@@ -16,8 +16,10 @@ sh /app/scripts/start_server.sh &
 coverage erase
 # spec validation
 python -m spec.validate
+# wait for the RE service to be up so integration tests can pass
+python -m relation_engine_server.utils.wait_for api
 # run importer/, relation_engine_server/, and spec/ tests
 coverage run --branch -m pytest
-# RE client tests
+# RE client tests - seemss like these tests were already run in the line above, why run them again?
```

Comment:
typo - `seems`

Reply:
Got it fixed locally, it'll show up in my next PR